features/link-rel-preload.yml

```diff
@@ -0,0 +1,14 @@
+name: '<link rel="preload">'
+description: The `rel="preload"` attribute for the `<link>` HTML element lets you declare fetch requests in the HTML's `<head>, specifying resources such as images or style sheets that your page will need very soon, which you want to the browser to start loading before main rendering begins.
```

Comment:
This one is described in rather different terms. Is it also a hint? Or does this feature have stronger guarantees about what's to happen when it's used?

Reply:
Interesting, where the other hints use RFC 2119 language in the spec, `preload` uses "will":

> The [preload](https://html.spec.whatwg.org/multipage/links.html#link-type-preload) keyword indicates that the user agent will preemptively [fetch](https://fetch.spec.whatwg.org/#concept-fetch) and cache the specified resource

After reading through all of that and a few guides, I think it's likely because it MUST not block `load` so it only guarantees that the fetches are queued with a priority flag... but can't guarantee other than that. It specifically says these are not actually loads, just requests-to-load.

I'll tweak to make that part clearer.